runtime/reconcilers/explore.go

```diff
@@ -127,10 +128,23 @@ func (r *ExploreReconciler) validateAndRewrite(ctx context.Context, self *runtim
 		return nil, fmt.Errorf("parent metrics view %q is invalid", spec.MetricsView)
 	}
 
-	// Add the access and field access security rules from the parent metrics view.
-	for _, rule := range mv.SecurityRules {
-		if rule.GetAccess() != nil || rule.GetFieldAccess() != nil {
-			spec.SecurityRules = append(spec.SecurityRules, rule)
+	if len(spec.SecurityRules) == 0 && len(mv.SecurityRules) > 0 {
+		for _, rule := range mv.SecurityRules {
+			if rule.GetAccess() != nil || rule.GetFieldAccess() != nil {
+				spec.SecurityRules = append(spec.SecurityRules, rule)
+			}
+		}
+	} else {
+		mv.SecurityRules = append(mv.SecurityRules, spec.SecurityRules...)
+		access := mergeConditionRules(mv.SecurityRules)
+		if access != nil {
+			spec.SecurityRules = []*runtimev1.SecurityRule{access}
+		}
+
+		for _, rule := range mv.SecurityRules {
+			if rule.GetFieldAccess() != nil {
+				spec.SecurityRules = append(spec.SecurityRules, rule)
+			}
```

Comment:
This is a bit dangerous because it will fail silently if a `RowFilter` or `FieldAccess` (or some fourth type of security rule we might define in the future) is used in the explore spec's security policy.

Consider explicitly looping over the explore spec's security rules and returning an error if one of them is not an `Access` rule